### DISCUSSION TITLE
The discussion message I want to visit from notification is the only one hidden behind "Show X more replies"

### DISCUSSION THREAD

**Discussion by ghost** [community/community#73240] [Discussions]
### Select Topic Area

Bug

### Body

`https://github.com/uBlockOrigin/uBlock-discussions/discussions/789#discussioncomment-7457569`

All the other sub-threads in this discussion have the latest messages visible and older ones hidden behind "Show X previous replies", and only the single sub-thread I want to see a message in it, have recent messages hidden behind "Show X more replies".

**Comment by dakshithadissanayaka** [comment]
The issue described by the user seems to be a bug in the GitHub Discussions interface. When a discussion thread has a large number of comments, the latest comments are hidden behind a "Show X more replies" button. However, in this particular case, only one sub-thread in the discussion has this issue, while the other sub-threads display the latest comments correctly.

This suggests that there may be a specific issue with the rendering of this particular sub-thread. It's possible that there is an error in the data for this sub-thread, or that the code responsible for rendering it is not handling the large number of comments correctly.

Here are some possible solutions to this issue:

1. **Reload the page:** Sometimes, a simple page reload can fix minor rendering issues. Try reloading the page to see if the latest comments are now visible.

2. **Clear browser cache:** Clearing your browser's cache can sometimes fix issues caused by corrupted or outdated data. Try clearing your browser cache and then reloading the page.

3. **Try a different browser:** If the issue persists, try using a different browser to access the discussion. This can help rule out any issues with your current browser's configuration or extensions.

4. **Report the bug:** If the issue persists across different browsers, report it to GitHub using their official support channels. Provide detailed information about the bug, including the URL of the discussion, the steps to reproduce the issue, and any relevant screenshots.

In the meantime, you can try using the "Show X previous replies" button to navigate to the latest comments in the affected sub-thread.

**Comment by Unmesh100** [comment]
It sounds like you've encountered a bug in the uBlockOrigin discussion interface. This issue where recent messages are hidden behind "Show X more replies" while other sub-threads display them correctly seems inconsistent and frustrating.

Would you mind reporting this bug on their GitHub page? Providing detailed information, like the specific sub-thread and any steps to reproduce the issue, would help the developers address it more effectively.
